modules/experimental-layers/src/addons/mapbox-map/mapbox-map.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * A simple mapbox-gl wrapper that works with deck props
+ * App is responsible for importing mapboxgl and passing it in as a prop:
+ *   import mapboxgl from 'mapbox-gl';
+ *   const map = new MapboxMap({mapboxgl, ... });
+ */
+const DEFAULT_STYLESHEET = 'https://api.tiles.mapbox.com/mapbox-gl-js/v0.44.2/mapbox-gl.css';
+
+export default class MapboxMap {
```

Comment:
> Import from react-map-gl like the lite module does?

I am not sure that makes sense. The JSON API itself has no React dependencies, and since is a live dependency (in contrast to the lite module which is conceptually a "script generator") so it doesn't seem reasonable for it to bring in react-map-gl as a dependency.